src/ngraph/op/experimental/shape_of.hpp

```diff
@@ -39,6 +39,11 @@ namespace ngraph
                     copy_with_new_args(const NodeVector& new_args) const override;
 
                 void validate_and_infer_types() override;
+
```

Comment:
It should be `set_is_foldable(bool value)`
This needs documentation.  What is it for? It looks like it is being used in such a way that it should be preserved in a cloning operation, which makes it like an attribute.

Reply:
Maybe rename it like `is_recursively_folded` which would has meaning that recursive folding has happened.

Reply:
Renamed `block_constant_folding` => `block_recurent_constant_folding`.
I would prefer to avoid introduce any new attributes because this will force me to upgrade the spec as well.
I added a comment to these two methods, is it enough for you?

Reply:
It's not an attribute; it doesn't affect the semantics of the op instance.